### PR TITLE
[RHACS] ROX-21700: Updating documentation for ACSCS certificate rotation

### PR DESCRIPTION
**Pull request by agantony**
Version(s):
4.3+

Issue:
https://issues.redhat.com/browse/ROX-21700

Link to docs preview:
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-apply.html
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.html
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/init-bundle-cloud-other-apply.html
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/cloud_service/installing_cloud_other/init-bundle-cloud-other-generate.html
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_ocp/init-bundle-ocp.html
- https://74831--ocpdocs-pr.netlify.app/openshift-acs/latest/installing/installing_other/init-bundle-other.html

SME review:
- [x] SME has approved this change.

Additional information:
- Merge into `rhacs-docs-main` and cherry-pick to:
    - `rhacs-docs-4.5`
    - `rhacs-docs-4.4`
    - `rhacs-docs-4.3`

### CODE REVIEW

**Review by ebensh — APPROVED** [review]

File: modules/create-resource-init-bundle.adoc [hunk 1]
```diff
@@ -90,8 +98,13 @@ $ kubectl create -f <init_bundle>.yaml \// <2>
...
 endif::kube[]
 
-.Next Step
-* Install {product-title-short} secured cluster services in all clusters that you want to monitor.
+ifdef::cloud-ocp-apply,cloud-other-apply[]
+.Verification
+
+* Restart Sensor to pick up the new certificates.
++
+For more information about how to restart Sensor, see "Restarting the Sensor container" in the "Additional resources" section.
```
[the remark this answers is not in the document]
Comment (agantony) [reply modules/create-resource-init-bundle.adoc:106]: I appreciate your feedback, but according to the documentation guidelines, we cannot add xref in modules.

**Review by kcarmichael08 — COMMENTED** [review]
The conditional text looks good! I just had one question that pertains to several assembly files about the "next steps" and "additional resources" sections.

**Comment by agantony** [comment]
> The conditional text looks good! I just had one question that pertains to several assembly files about the "next steps" and "additional resources" sections.

@kcarmichael08 Thanks for the feedback. I have resolved all the comments. If everthing else looks good to you, please go ahead and merge the PR.

**Review by kcarmichael08 — COMMENTED** [review]
LGTM

**Pull request merged**